app/views/admin/tickets/_form.html.haml

```diff
@@ -14,4 +14,5 @@
       = f.input :price
       = f.input :price_currency, as: :select, class: 'form-control', collection: ['USD', 'EUR', 'GBP', 'INR', 'CNY'], include_blank: false
       %p.text-right
-        = f.action :submit, as: :button, button_html: { class: 'btn btn-primary' }
+      = f.input :registration_ticket, hint: 'A registration ticket is with which user register for the conference.'
+      = f.action :submit, as: :button, button_html: { class: 'btn btn-primary' }
```

Comment:
this line used to be inside  `%p.text-right` and it is not there anymore :confused: